collections/src/main/scala/strawman/collection/package.scala

```diff
@@ -4,6 +4,8 @@ import scala.{Any, AnyVal, Array, Boolean, Char, IllegalArgumentException, Index
 import scala.Predef.{String, ArrowAssoc}
 
 package object collection extends LowPriority {
+  @deprecated("Use Iterable instead of Traversable", "2.13.0")
+  type Traversable[+X] = Iterable[X]
```

Comment:
Maybe we still need a type alias for backward compatibility?

Reply:
WDYM? This is the type alias.

Reply:
Oops, I must have been color blind when I reviewed the PR…